c2cgeoportal/lib/dbreflection.py

```diff
@@ -161,16 +161,20 @@ def get_table(tablename, schema=None, session=None):
             "ignore",
             "Did not recognize type 'geometry' of column",
             SAWarning)
+        args = [tablename, metadata]
+        if primary_key is not None:
+            # Ensure we have a primary key to be able to edit views
+            args.append(Column(primary_key, Integer, primary_key=True))
         table = Table(
-            tablename, metadata,
+            *args,
             schema=schema,
             autoload=True,
-            autoload_with=engine,
+            autoload_with=engine
```

Comment:
? for me it's better with a coma...

Reply:
Strangely I get a 500 error with the coma.

Reply:
What you wish is not supported in python prior to 3.6.
See https://stackoverflow.com/questions/33350454/syntaxerror-with-passing-kwargs-and-trailing-comma.

Reply:
I never see that but it's true ...